config/initializers/constants.rb

```diff
@@ -0,0 +1,2 @@
+# Hold Google maps API key
+GOOGLE_MAPS_API_KEY = Rails.application.secrets.GOOGLE_MAPS_API_KEY
```

Comment:
Added a newline now.

Just curious, what is the issue you faced in production. Could hold me in good stead for the future 🛡